app/src/main/res/values-om-rET/strings.xml

```diff
@@ -511,8 +527,8 @@
 
   <!-- Contact patient -->
   <string name="contactpatient_agreed_to_visit">Dhukkubsataa \“dhufuudhaaf kan waliigale” jedhii milikkiteessi.</string>
-  <string name="contactpatient_remind_call_later">Booddee na yaadachiisi…</string>
-  <string name="contactpatient_remove_from_list">Tarree  irra darbeerraa kaasi…</string>
+  <string name="contactpatient_remind_call_later">Booddee na yaadachiisi...</string>
+  <string name="contactpatient_remove_from_list">Tarree  irra darbeerraa kaasi...</string>
```

Comment:
Can you tell me why in some places we have `. . .` and in some places we've changed it to `...` ?

Reply:
`...` is incorrect. What we should be using is the ellipsis character (`…`).

The translators tend to make this mistake often. We have a GitHub action that will run on pushes to the branch that will correct this specific error.

Reply:
Makes sense now. Thanks